README.md

```diff
@@ -1,5 +1,5 @@
 # privateGPT
-Ask questions to your documents without an internet connection, using the power of LLMs. 100% private, no data leaves your execution environment at any point. You can ingest documents and ask questions without an internet connection!
+Ask questions about your documents without an internet connection, using the power of LLMs. 100% private; no data leaves your execution environment at any point. You can ingest documents and ask questions without an internet connection!
```

Comment:
Please check if this needs to changed back to "to"

Reply:
I think "to" in this context is that the LLM assumes your document as a "character" so you are asking questions to your document

Reply:
So, I'll make some quick changes.